docs/solution-architecture/accessControlPolicies.md

```diff
@@ -78,7 +78,7 @@ various resources on which a user can perform actions:
 | toevoegen_bag_object                                 |                                                    ✅<br/>_(zaak open)_                                                    |             |                                 ✅                                  |           |
 | starten_taak                                         |                                        ✅<br/>_(afhankelijk van CMMN status zaak)_                                         |             |                                                                    |           |
 | vastleggen_besluit                                   |                             ✅<br/>_(zaak open, niet in intake, zaaktype heeft besluittypen)_                              |             |                                                                    |           |
-| verlengen_doorlooptijd                               |                                    ✅<br/>_(zaak open, niet heropend, niet opgeschort)_                                    |             |                                                                    |           |
+| verlengen_doorlooptijd                               |                                    ✅<br/>_(zaak open, heropend, opgeschort)_                                    |             |                                                                    |           |
```

Comment:
At the moment this is implemented like:
```
default verlengen_doorlooptijd := false
verlengen_doorlooptijd {
    behandelaar.rol in user.rollen
    zaaktype_allowed == true
    zaak.open == true
    zaak.opgeschort == false
}
```

Which means opened and `niet opgeschort`
If we want to change it we need to touch policies and tests too.

Reply:
That's strange, according to Karin it is working this way at the moment. I will take a look

Reply:
Yes exactly. We cannot simply update the `accessControlPolicies.md` document. Rather this is a reflection of the policy implementation of our code. So it can only be changed together with changing the policy implementation (and related tests).

Reply:
The reality at the moment and the whish from the workgroup is the change I try to make in the list. Why is that an issue?
What's written is wrong.
I will ask Karin to write a task for it

Reply:
I am a bit puzzled, before we had a list in the wiki and the code had to follow that. Now it is in github it is the other way around?